Fix vale (#41)
canonical/netbox: charm/docs/explanation/charm-architecture.md

- Redis integration
- Django migrations
- Ingress integration
- COS (Prometheus metrics and Loki logs for `gunicorn`).

For the static assets, `gunicorn` is used with the help of the [WhiteNoiseMiddleware](https://whitenoise.readthedocs.io/en/stable/index.html).

There is only one container for each unit of NetBox that runs the following
services managed by Pebble:

> - `django`. Runs `gunicorn`.
> - `cron`. `Cron` service that runs management commands for `housekeeping` and `syncdatasource`.
> - `statsd_exporter`. To expose `gunicorn` metrics.

Reviewer: ```suggestion
- `django`: runs `gunicorn`.
- `cron`: periodically runs the `housekeeping` and `syncdatasource` management commands.
- `statsd_exporter`: exposes `gunicorn` metrics.
```
